React/Base/RCTPerformanceLogger.h

```diff
@@ -7,6 +7,7 @@
 
 #import <Foundation/Foundation.h>
 
+// Keep this in sync with _labelsForTags
```

Comment:
I think we'd better to remove these missing labels in `_labelForTags`? because we don't use these enums at all.

Reply:
Yup, +1